Sources/GRPC/AsyncAwaitSupport/GRPCAsyncServerHandler.swift

```diff
@@ -153,13 +158,10 @@ extension GRPCAsyncServerHandler {
 @usableFromInline
 internal final class AsyncServerHandler<
   Serializer: MessageSerializer,
-  Deserializer: MessageDeserializer
->: GRPCServerHandlerProtocol {
-  @usableFromInline
-  internal typealias Request = Deserializer.Output
-  @usableFromInline
-  internal typealias Response = Serializer.Input
-
+  Deserializer: MessageDeserializer,
+  Request: Sendable,
+  Response: Sendable
```

Comment:
Why do we have an explicit Request and Response here? Cant we say 
```swift
where Deserializer.Output == Sendable, Serializer.Input == Sendable
```

Reply:
Good idea, we can do that.